packages/commons-test/src/testConfig.ts

```diff
@@ -0,0 +1,9 @@
+import { EmailManagerConfig } from "pagopa-interop-commons";
+import { z } from "zod";
+
+export const EmailManagerConfigTest = EmailManagerConfig.and(
+  z.object({
+    smtpHTTPPort: z.number().optional(),
```

Comment:
```suggestion
    smtpPort: z.number().optional(),
```

Reply:
It's actually a different port: it's the HTTP port to call the mailpit API (the test email server) used just in tests - i renamed it to make it clearer